handbook/engineering/security/index.md

```diff
@@ -4,9 +4,51 @@ We think that security is an enabler for the business. Sourcegraph is committed
+
+**We share our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph cloud**
+
+  - Problem and rationale: While Sourcegraph cloud is in an good, and improvine security state, we currently do not have the right level of visibility. By starting with a private repository that contains no secrets, we can build and deploy the visibility technologies and processes that allow us to build a holistic view of the environment.  This serves as a proxy for small company repository use.
+  - Planned work:
+    - Run ongoing vulnerability scans of the containers including vulnerability remediation
+    - Deploy centralized security logging
+    - Ship container logs to the logging destination
+    - Begin documenting policies
+  - Definition of success: Frequent container vulnerability scans are being run, and stored for future a audit. Fixes to the containers or their underlying service configurations have made it into the product release cycle.  Logs of new container deployments are available for search in a centralized logging tool.
+
+**We share a private repository containing encrypted secrets**
+
+  - Problem and rationale: Customers will have repositories containing encrypted secrets, using an [ansible vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) or gpg encrypted files. Trusting these repositories to external systems requires the added degree of care and concern, expected in security conscious small and medium enterprise (SME). To support those needs, not only is enhanced visibility into our own systems required on our side, it will be contractually obligated, or at a minimum part of security questionnaires.
+  - Planned work:
+    - Run ongoing vulnerability scans of underlying workloads including vulnerability remediation
+    - Deploy host-based intrusion detection (HIDS)
+    - Centralize cloud and workload logs in a single location
+    - Add support for audit and access logging
+    - Normalize security and application logs
+    - Create dashboards and alerts for events of interest
+    - Extend policy documentation
+  - Definition of success: We deploy a private repository containing encrypted secrets to Sourcegraph cloud, complete with a baseline for security monitoring. We are able to validate access to underlying hosts and the application in centralized logging.
+
+**Host a capture the flag for Sourcegraph.com**
+
+  - Problem and rationale: Running a world class service capable of hosting private repositories is about more than application security. As examples, we need visibility into data loss, processes around incident response need documentation, and defensive measures for the internal actor need to be expanded upon. The proxy enhances the addressed security concerns that a medium enterprise would require.
```

Comment:
```suggestion
  - Problem and rationale: Running a world class service capable of hosting private repositories is about more than application security. As examples, we need visibility into data loss, processes around incident response need documentation, and expanded defensive measures against internal threat actors. The proxy enhances the addressed security concerns that a medium enterprise would require.
```

Reply:
Modified your text to and reworked. Perhaps this is better?